gym/envs/tests/test_determinism.py

```diff
@@ -75,5 +74,8 @@ def assert_equals(a, b, prefix=None):
             assert_equals(v_a, v_b)
     elif isinstance(a, np.ndarray):
         np.testing.assert_array_equal(a, b)
+    elif isinstance(a, tuple):
+        for elemFromA, elemFromB in zip(a, b):
```

Comment:
We use underscore rather than camel everywhere else — we're generally not super strict about style, but please try to match the surrounding code